app/services/Config.scala

```diff
@@ -112,6 +112,8 @@ sealed trait Config {
   def frontendBucketWriteRole: Option[String] = None
   def auditingKinesisWriteRole: Option[String] = None
   def enableAuditStreaming: Boolean = true
+
+  def tagSearchPageSize = 25
```

Comment:
I think it should be in some configuration file instead of just being passed in by the component and the configuration gets passed in by the server to that's why.